src/components/LibrariesListItem.tsx

```diff
@@ -55,7 +55,9 @@ export default class LibrariesListItem extends React.Component<LibrariesListItem
     let style = this.state.color;
     let { name } = this.props.library.basic_info;
     return(
-      <li><Panel style={style} headerText={name} content={this.content()} /></li>
+      <li>
+        <Panel id={name.replace(/ /g, "")} style={style} headerText={name} content={this.content()} />
```

Comment:
Is the ID exposed to users/screen readers?  If so, doing `replace` with dashes or underscores will be a lot easier to read.

Reply:
No, IDs don't get exposed to users or screenreaders. I'll uses dashes.